### PR TITLE
Counterspell Melbourne

### PR DESCRIPTION
**Pull request by Jayx2u**
_nice_

### CODE REVIEW

File: src/components/Locations.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ export default function Locations() {
     "Austin, TX",
     "San Francisco, CA",
     "Burlington, VT",
-    "Oshkosh, WI"
+    "Oshkosh, WI",
+    "Melbourne, VIC, AUS"
```
Comment (samdev-7) [inline src/components/Locations.js:17]:
```suggestion
    "Oshkosh, WI"
```
Don't change this yet! We'll update it after organizer registrations close.

File: src/index.js [hunk 2]
```diff
@@ -23,7 +25,10 @@ root.render(
           Replace `/example-city` and `ExampleCity` with the name of your city.
           E.g. Ottawa, ON -> `/ottawa` and `Ottawa`.
         */}
-        {/* <Route path="/example-city" element={<ExampleCity />} /> */}
+        {/* <Route path="/example-city" element={<ExampleCity />} /> */
+
+          <Route path="/Melbourne" element={<Melbourne />} />
+        }
```
Comment (samdev-7) [inline src/index.js:31]:
```suggestion
        {/* <Route path="/example-city" element={<ExampleCity />} /> */}

          <Route path="/melbourne" element={<Melbourne />} />
```

**Review by samdev-7 — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import App from "./pages/App";
 import NotFound from "./pages/NotFound";
 
+import Melbourne from "./pages/Melbourne"
+
```
Comment (samdev-7) [inline src/index.js:10]: I'm being picky here, but can you move this below the `// import ExampleCity from "./pages/ExampleCity";` comment?

File: src/index.js [hunk 4]
```diff
@@ -25,6 +27,8 @@ root.render(
         */}
         {/* <Route path="/example-city" element={<ExampleCity />} /> */}
```
Comment (samdev-7) [inline src/index.js:29]:
```suggestion
```
Remove the extra line

**Review by samdev-7 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 3 above)
Comment (Jayx2u) [reply src/index.js:10]: It's all good :>

File: src/index.js [hunk 5]
```diff
@@ -6,13 +6,13 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import App from "./pages/App";
 import NotFound from "./pages/NotFound";
 
-import Melbourne from "./pages/Melbourne"
-
 // To add your own page, uncomment the following import statement.
 // Replace `ExampleCity` with the name of your city.
 // E.g. Ottawa, ON -> `Ottawa`.
 // import ExampleCity from "./pages/ExampleCity";
```
Comment (samdev-7) [inline src/index.js:13]:
```suggestion
```
Sorry, being picky again. Just remove this empty line and we should be set!

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**